lib/improver/threshold.py

```diff
@@ -32,6 +32,8 @@
 
 
 import numpy as np
+import iris
+from cf_units import Unit
 
 
 class BasicThreshold(object):
```

Comment:
`process` method should take a copy of input cube rather than modifying it. While out of scope of the ticket, this would avoid errors in the future.

Reply:
Done.

Reply:
This is a good point, and should apply to lots of plugins